.env

```
@@ -0,0 +1,67 @@
+# Database Configuration
+MONGODB_URI=mongodb://localhost:27017/defiendete-mx
+
+# For production, use MongoDB Atlas or similar:
+# MONGODB_URI=mongodb+srv://username:password@cluster.mongodb.net/defiendete-mx?retryWrites=true&w=majority
+
+# JWT Secrets (for authentication)
+JWT_SECRET=your-super-secret-jwt-key-change-this-in-production
+JWT_REFRESH_SECRET=your-super-secret-refresh-key-change-this-in-production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_REFRESH_SECRET key should go before the JWT_SECRET key

(UnorderedKey)